### ISSUE TITLE
CPT custom archive slug = not listed in sitemap

### ISSUE THREAD

**Issue by quadcom** [sybrew/the-seo-framework#596]
**Describe the bug**
I have two custom post types FAQ and Reviews. the FAQ post type archive page does not have a custom slug defined.

The FAQ PT archive page does not have a custom slug defined
![image](https://user-images.githubusercontent.com/7085931/142652563-9dced39b-ddb6-4c32-9157-676035bf67cc.png)

The Reviews PT archive page does have an archive page custom slug defined.
![image](https://user-images.githubusercontent.com/7085931/142652654-a411ea75-d8ed-4f0e-ae14-e05d55681c96.png)

I have noticed that the post type with the custom slug does not show up in the site map TSF generates.
![image](https://user-images.githubusercontent.com/7085931/142653366-d1ccddd9-a6a0-456a-94fb-e36dce56f17f.png)

**How to reproduce the bug**
Simply provide a custom slug for the post type

**Expected behavior**
I would expect that the custom post type archive page to be listed in the sitemap.

**Screenshots**
Included inline

**Additional context**
Running latest v4.20

**Comment by sybrew** [comment]
For future reference: Archives aren't added to the sitemap of TSF.

https://tsf.fyi/kb/a/21

**Comment by quadcom** [comment]
I pulled the post as I discovered that I can pseudo-force the archives to show in the sitemap. I create a blank page using the same slug as the archive. It shows up in the sitemap and the archive steals the content for the slug. I needed the meta info for archives as the meta assigned to the page does not transfer over to the archive content that gets displayed.

But since sitemaps are going the way of the DoDo soon.......
